### DISCUSSION TITLE
BRP072C42 Controller

### DISCUSSION THREAD

**Discussion by lochstar** [revk/xESP32-Faikout#544] [General]
Hi all,

I've just successfully replaced my pair of `BRP072C42` controllers with Faiken.

I've added the details of my 2 units, `FTXV60WVMA` and `FTXV35UVMA` to the Wiki.

Most functions all work fine, however the special modes all immediately toggle off. I can set these via the physical remote.

Is there anything I can do to investigate further to assist the project? I have these 2 controllers that I was planning on reselling, but if they could have a use I am willing to donate.

I've attached a PDF with some info about this particular controller.

[BRP072A42 Transition to BRP072C42 (Mobile Controller).pdf](https://github.com/user-attachments/files/17708884/BRP072A42.Transition.to.BRP072C42.Mobile.Controller.pdf)

**Comment by revk** [comment]
Make sure in beta release anyway as easier for me to debug...

There is a dump setting which sends debug to MQTT. That may be helpful if you catch it happening. There is also a debug setting which logs even more.

**Reply by lochstar** [reply]
Yep, I'm on beta for both.

I'll enable the debug setting and try toggling the modes and see what comes up.
